src/app.js

```diff
@@ -1,17 +1,33 @@
 import express from 'express'
 import userRouter from './router/userRouter'
+import internalRouter from './router/internalRouter'
 import handleValidationError from './error/handleValidationError'
 import handleError from './error/errorHandler'
+import logMethodInvocation from './logger/infoLoggerMidelware'
+import { generalLogger as logger } from './logger/logger'
 
 const PORT = process.env.PORT || 8182
 const app = express()
 
 app.use(express.json())
 app.use(express.urlencoded({ extended: false }))
 
+app.use(logMethodInvocation)
+
 app.use('/api/users', userRouter)
+app.use('/api/internal', internalRouter)
```

Comment:
I think it is bad practice to be able to affect on your internal server work from external api

Reply:
Why? How can I change debug mode in runtime?

Reply:
Discussed